pkg/lib/web/image.go

```diff
@@ -303,3 +305,7 @@ func (a ImageDescriptor) viewByPath(resources []resource.ResourceFile, path stri
 		Data: bytes,
 	}, nil
 }
+
+func (a ImageDescriptor) GetSizeLimit() int {
+	return a.SizeLimit
```

Comment:
Give it a default? (100KB)

Reply:
Is it I check if SizeLimit is empty value, i.e. 0, then return default value 100kb?

Reply:
Yes